example-tags/tag-init.json

```diff
@@ -1,5 +1,6 @@
 {
     "Existing Tags": [ "Mathe", "Deutsch", "Englisch", "Latein", "Physik", "Biologie", "Geschichte", "Erdkunde"
     ],
-    "Existing Users": [ "testuser", "Ash", "Markus", "Nils", "Theo" ]
+    "Existing Users": [ "testuser", "Ash", "Markus", "Nils", "Theo" ],
+    "File Types": [ "Picture", "Text", "Audio", "Video" ]
```

Comment:
You forgot the most imporntant: PDF, maybe also add "Markdown" / "MD".

Reply:
Yeah, I'd suggest Document.. for pdfs, and markdowns i's usually also consider Documents, so let's just replace the text with document?

Reply:
Sure!

Reply:
none